darts/ad/scorers/kmeans_scorer.py

```diff
@@ -0,0 +1,216 @@
+"""
+KMeansScorer
+-----
+
+KMeans Scorer (k-means clustering) [1]_.
+The implementations is wrapped around `scikit-learn
+<https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html>`_.
+
+References
+----------
+.. [1] https://en.wikipedia.org/wiki/K-means_clustering
+"""
+
+from typing import Optional, Sequence
+
+import numpy as np
+from sklearn.cluster import KMeans
+
+from darts.ad.scorers.scorers import FittableAnomalyScorer
+from darts.logging import raise_if_not
+from darts.timeseries import TimeSeries
+
+
+class KMeansScorer(FittableAnomalyScorer):
+    def __init__(
+        self,
+        window: Optional[int] = None,
```

Comment:
Is a window of None equivalent to 1? If yes, set the default value of 1 here for clarity (it's OK to use default values for non-mutable types :) )